### PR TITLE
Remove dependency (in cloud workers) on gsutil.

### PR DESCRIPTION
**Pull request by sorensenjs**
PTAL - still testing, but I think this will move us significantly towards python3 compatibility.
gsutil has no forward path at this moment.

### CODE REVIEW

File: wikiconv/ingest_revisions/dataflow_main.py [hunk 1]
```diff
@@ -23,78 +24,56 @@
...
-                   --ingestFrom=wikipedia --download --language=YourLanguage\
-                   --dumpdate=YourDumpdate --cloudBucket=YourCloudBucket\
-                   --project=YourGoogleCloudProject --bucket=TemporaryFileBucket]
-  - local: Tests the pipeline locally, run the code with
-           [python dataflow_main.py --setup_file ./setup.py\
-            --ingestFrom=local --localStorage=YourLocalStorage --testmode\
-            --output=YourOutputStorage --project=YourGoogleCloudProject\
-            --bucket=TemporaryFileBucket]
+  - wikipedia: performs the downloading job from Wikipedia, run with: [python
+    dataflow_main.py --setup_file ./setup.py --ingestFrom=wikipedia --download
```
Comment (iislucas) [inline wikiconv/ingest_revisions/dataflow_main.py]: line breaks here will make copy paste really horrible.

Comment (sorensenjs) [reply wikiconv/ingest_revisions/dataflow_main.py]: but backslashes in strings cause lint complaints - will try to find a trade off.

**Comment by sorensenjs** [comment]
Confirming this change successfully passed the first two steps. All tests also pass, but that doesn't mean much at present. I plan to submit and continue with the next step. (I'm running on the quite modest latin wikipedia and considering using one of the very smallest wikipedias as a checked in end2end test case.)

I also intend to send a pull request documenting the important step of setting up the environment to point to the service account json authorization file.

**Comment by iislucas** [comment]
Sounds great, thanks for all this work Jeff, and for checking in test,
consider checking in just a fragment of the data too? E.g. one month of
data.

**Pull request merged**